resources/lang/en/general.php

```diff
@@ -120,6 +120,10 @@
     'firstname_lastname_underscore_format' => 'First Name Last Name (jane_smith@example.com)',
     'lastnamefirstinitial_format' =>  'Last Name First Initial (smithj@example.com)',
     'firstintial_dot_lastname_format' => 'First Initial Last Name (j.smith@example.com)',
+    'firstname_lastname_display'	=> 'First Name Last Name (Jane Smith)',
+    'lastname_firstname_display'	=> 'Last Name First Name (Smith Jane)',
+    'name_display_format'	=> 'Name Display Format',
+    'name_display_help'	    => 'This is generally only used in countries where it is common to write  ',
```

Comment:
I don't think `name_display_help` is being used?

Reply:
100% correct - I'll pull it. I wanted to be clearer, but since it defaults to first name last name, I think it kind of explains itself